source/mbed_platform.cpp

```diff
@@ -69,12 +93,25 @@ void sleepFromUntil(tick_t now, tick_t until){
     } else {
         const uint32_t next_int = lp_ticker_get_compare_match();
 
+#if YOTTA_CFG_MINAR_STATS
+        // update stats
+        tick_t now = lp_ticker_read();
+        total_active += now - last_wakeup;
```

Comment:
```tick_t``` and ```lp_ticker_read()``` are both ```uint32_t``` so the subtraction is overflow neutral as long as both ```now``` and ```last_wakeup``` belong to the same ```epoch```, which they will since the overflow handler will wake up minar before that happens.